server/src/com/treode/server/Schema.scala

```diff
@@ -13,4 +15,8 @@ class Schema (mapping: HashMap [String, Long]) {
         => id.toString.getTableId
         case None =>
         throw new BadRequestException (s"Bad table ID: $s")
-    }}}
+    }}
+    
+  def == (schema: Schema): Boolean = {
```

Comment:
Generally we define `hashCode` and `equals`, not `==`. But, you could instead declare this as:

```
case class Schema (map: Map [String, Long])
```

Then: 1) map can be access as a property, 2) they compiler defines `hashCode` and `equals` for you.